rfcs/text/0017-remove-log-original.md

```diff
@@ -72,10 +82,6 @@ The removal of `log.original` will be considered a breaking change since the fie
 
 As a breaking change, this would require timely communication to the Elastic Community.
```

Comment:
Let's note the resolution taken to address this concern.

Can you describe the work you've done to communicate this change to internal stakeholders, @djptek?

Reply:
added detail